crates/e2e/src/client.rs

```diff
@@ -573,6 +573,50 @@ where
         value: E::Balance,
         storage_deposit_limit: Option<E::Balance>,
     ) -> Result<CallResult<C, E, <M as InkMessage>::ReturnType>, Error<C, E>>
+    where
+        M: InkMessage,
+        <M as InkMessage>::ReturnType: scale::Decode,
+    {
+        let call_result = self
+            .call_raw(
+                signer,
+                account_id,
+                contract_call,
+                value,
+                storage_deposit_limit,
+            )
+            .await?;
+        let value: <M as InkMessage>::ReturnType = scale::Decode::decode(
+            &mut call_result.value.as_ref(),
+        )
+        .unwrap_or_else(|err| {
+            panic!(
+                "decoding dry run result to ink! message return type failed: {}",
+                err
+            )
+        });
+
+        Ok(CallResult {
+            value,
+            dry_run: call_result.dry_run,
+            events: call_result.events,
+        })
+    }
+
+    /// Executes a `call` for the contract at `account_id`.
+    ///
+    /// Returns when the transaction is included in a block. The return value
+    /// contains all events that are associated with this transaction.
+    ///
+    /// The raw bytes of the
```

Comment:
Missing rest of the

Reply:
Removed this fn now in favour of making `call` succeed even if a decoding error